### PR TITLE
api module

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

**Comment by jfly** [comment]
https://github.com/NixOS/nixpkgs/issues/254552

File: nixos-tests/api.nix [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      services.pr-tracker-api.localDb = true;
+    };
+
+    testScript = ''
+      pr_tracker_api.start()
+
+      pr_tracker_api.wait_for_unit("pr-tracker-api.service")
+
+      # https://github.com/molybdenumsoftware/pr-tracker/issues/9
+      pr_tracker_api.wait_until_succeeds("curl --fail --connect-timeout 2 ${urlRoot}/api/v1/healthcheck", timeout=60)
```
Comment (RaitoBezarius) [inline nixos-tests/api.nix:55]: Just do a `wait_for_open_port` in the meantime.

Comment (mightyiam) [reply nixos-tests/api.nix]: Thank you. We did [a thing](https://github.com/molybdenumsoftware/pr-tracker/pull/28), instead.

**Comment by jfly** [comment]
Commenting so that some other PRs are the least recently updated. They are used as test fixtures.

**Pull request merged**